bower.json

```diff
@@ -22,7 +22,7 @@
     "map"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.3"
+    "polymer": "Polymer/polymer#^2.0.0"
```

Comment:
@sheideman Please keep Polymer 1.x support and use hybrid variants:
```
  "dependencies": {
    "polymer": "Polymer/polymer#1.9 - 2"
  },
  "devDependencies": {
    "iron-component-page": "PolymerElements/iron-component-page#1 - 2",
    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0",
    "iron-flex-layout": "PolymerElements/iron-flex-layout#1 - 2"
  },
  "variants": {
    "1.x": {
      "dependencies": {
        "polymer": "Polymer/polymer#^1.9"
      },
      "devDependencies": {
        "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
        "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
        "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0"
      }
    }
  }
```

I've added `webcomponentsjs` and `iron-flex-layout` because they are required for the demo: https://github.com/ebidel/geo-location/blob/master/demo/index.html#L6-L9.